pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - 'packages/**'
```

Comment:
Shouldn't `packages` allow to have the same dev dependencies everywhere? 🤔

Reply:
pnpm doesn’t use dependency hoisting:

When installing dependencies with npm or Yarn Classic (v1), all packages are hoisted to the root of the modules directory. As  a result, source code has access to dependencies that are not added as dependencies to the project.
By default, [pnpm](https://pnpm.io/motivation#creating-a-non-flat-node_modules-directory) uses symlinks to add only the direct dependencies of the project into the root of the modules directory.

In practice it means that if you have a package A that imports a package B (import something from 'B') but doesn’t explicitly specify B in the dependencies or devDependencies, then the execution will fail.